lib/Gemfile

```diff
@@ -22,16 +22,16 @@ gem "binary_struct",        "~> 2.0",        :require => false
 gem "excon",                "~>0.40",        :require => false
 gem "ezcrypto",             "=0.7",          :require => false
 gem "facade",               "~>1.0.5",       :require => false  # Used by util/pathname2.rb
-gem "ffi",                  "~>1.9.3",       :require => false
+gem "ffi",                  "=1.9.8",        :require => false
```

Comment:
After discussing with @Fryguy, bumping the version number to 1.9.8 is fine but we should keep the `~>1.9.8` requirement to allow patch releases.